core/src/main/java/org/akaza/openclinica/service/JobTriggerService.java

```diff
@@ -111,7 +111,7 @@ public void triggerJob(){
 						ruleSetService.runRulesInBulk(ruleSetBeans, false, currentStudy, ub, true);
 					} else {
 						// Event Specific Rule
-						System.out.println("*** Event Specific Rule ***");
+						logger.info("*** Event Specific Rule ***");
```

Comment:
any reason for setting this to info as opposed to debug?

Reply:
I have changed according to your comments